Code_Exercises/Exercise_18_Local_Memory_Tiling/README.md

```diff
@@ -11,8 +11,7 @@ tiles according to work-groups in order to compare the performance difference.
 ### 1.) Use local memory
 
 Allocate local memory for the kernel function by creating a local `accessor`,
-you can do this by creating an `accessor` with the `access::target::local`
-access target template parameter.
```

Comment:
Here I think we can just remove the rest of the sentence entirely rather than repeating ourselves

Reply:
Yep, removed.